apps/meteor/ee/server/api/licenses.ts

```diff
@@ -8,13 +8,13 @@ import { hasPermissionAsync } from '../../../app/authorization/server/functions/
 
 API.v1.addRoute(
```

Comment:
Shouldn't we remove this endpoint instead of updating it?

Reply:
I thought about this too, but decided to updated this endpoint regardless of our plans to remove it in the next release. So that even if we delay the removal by any reason, this would have a better permission validation
wdyt? :thinking: I could remove this update too

Reply:
I agree with Kevin, since it's already deprecated and the release that is planned for it to be removed is 7.0.0, we should remove it instead.

Reply:
I won't touch this endpoint in this PR, then. Let's wait for another one to remove it